classes/book.rb

```diff
@@ -1 +1,15 @@
 # book class
```

Comment:
- Class `Book` is a child of class `Item` as from the UML class diagram. Please show this kind of relationship